### ISSUE TITLE
Search component - When all ages are selected it looks very weird... 

### ISSUE THREAD

**Issue by GaliShapira** [ji-devs/ji-cloud#2878]
<img width="928" alt="Screen Shot 2022-06-29 at 11 47 51" src="https://user-images.githubusercontent.com/88834002/176438334-46796fbb-46a7-47f5-8bfb-364f8c1620ce.png">
<img width="727" alt="Screen Shot 2022-06-29 at 11 44 18" src="https://user-images.githubusercontent.com/88834002/176438353-8a484fb4-64a6-4eae-af5a-f71e64935b6a.png">

**Comment by corinnewo** [comment]
@johnnynotsolucky Please change it so that if there are multiple selections for the age, it lists them until the end of the field, and then has a '...'
This is urgent to fix as we are giving important demos this week! Thanks!

**Comment by johnnynotsolucky** [comment]
@GaliShapira @corinnewo this looks like a bug affecting Chrome. Safari and Firefox seem to be unaffected, and render the ages correctly.

![image](https://user-images.githubusercontent.com/4161106/178205661-5c1b178f-befc-4c6c-a965-70162614235c.png)

**Comment by corinnewo** [comment]
Can you fix it?

**Comment by johnnynotsolucky** [comment]
Yes, I'm just trying to make sure that I don't break it everywhere else the element is used.
